packages/frontend/src/components/profile_pic.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import { useEffect } from "react";
+import { Tag } from "./tags";
+
+interface Info {
+  sportName: Sport;
+  selected: boolean;
+  abilityLevel: string;
+  onClick: () => void;
+}
+
+interface SportInfo {
```

Comment:
Collides with one of our type names and only seems to be used once, where `image` is added. So you could move it into one interface called `Props`